src/Middleware/Diagnostics/src/DeveloperExceptionPage/Views/ErrorPage.cshtml

```diff
@@ -315,12 +315,11 @@
                     {
                         <tr>
                             <td><span title="@(metadata.GetType().FullName ?? string.Empty)">@metadata.GetType().Name</span></td>
-                            <td>@HtmlEncodeAndReplaceLineBreaks(metadata.ToString() ?? string.Empty)</td>
-                        </tr>
-
-                        <tr>
-                            <td><span title="@(metadata.GetType().FullName ?? string.Empty)">@metadata.GetType().Name</span></td>
-                            <td>@(metadata.ToString() ?? string.Empty)</td>
+                            <td>
+                                @{
```

Comment:
Oops! Sorry, I was trying to get feedback on which way was better to do. https://github.com/dotnet/aspnetcore/pull/52668#discussion_r1428029925

Since some times has been passed, I forgot about it!